### DISCUSSION TITLE
uinput fix on Fedora Atomic

### DISCUSSION THREAD

**Discussion by SaltScratch** [AntiMicroX/antimicrox#966] [Show and tell]
ref https://github.com/AntiMicroX/antimicrox/wiki/Open-uinput-error

The above instructions won't work on Fedora Atomic distributions (eg. Silverblue) because the /usr/lib folder is read-only. Instead create the file in **/usr/local/lib/udev/rules.d**. You may need to create the udev and rules.d folders first.

**Comment by pktiuk** [comment]
Thank you for info. I addded it to the Wiki.
